.github/workflows/_linux-build.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
       - name: Calculate docker image
         id: calculate-docker-image
-        uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
+        uses: pytorch/test-infra/.github/actions/calculate-docker-image@wait-for-docker-build
```

Comment:
Testing https://github.com/pytorch/test-infra/pull/6013 that waits for Docker build